### PR TITLE
Create `build-simple-sample.yml` workflow

### PR DESCRIPTION
**Pull request by hoc081098**
Build `:app:simple` when pushing to main or opening PR to main

### CODE REVIEW

**Comment by hoangchungk53qx1** [comment]
nice ✌️ ,

File: .github/workflows/build-simple-sample.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build simple sample
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./sample/simple
```
Comment (gabrielittner) [inline .github/workflows/build-simple-sample.yml:15]: We could alternatively use ./gradlew -p sample/simple ...` below and avoid the `../..`

File: .github/workflows/build-simple-sample.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      - name: Install Android SDK
+        run: ../../.github/android-sdk.sh
+
+      - uses: gradle/gradle-build-action@v2
+
+      - name: Build with Gradle
+        run: ./gradlew :app:simple:assembleDebug --stacktrace
+
+      - name: Check code style
+        run: ../../.kotlinw ../../.kts/ktlint.main.kts --fail-on-changes
```
Comment (gabrielittner) [inline .github/workflows/build-simple-sample.yml:39]: The script finds all kt files regardless of Gradle projects, so the existing pr check should be enough

**Review by gabrielittner — APPROVED** [review]
Thanks, this is great. Left 2 minor comments but still going ahead with merging this so that it can run

**Pull request merged**